app/src/components/market/common/outcome_table/index.tsx

```diff
@@ -163,7 +165,9 @@ export const OutcomeTable = (props: Props) => {
           <TDStyled textAlign={TableCellsAlign[2]}>
             <TDFlexDiv textAlign={TableCellsAlign[2]}>
               {currentPriceFormatted}{' '}
-              {showSharesAndPriceChange && <NewValue outcomeIndex={outcomeIndex} value="1.23" />}
+              {showSharesAndPriceChange && currentPriceFormatted !== newPrice && (
+                <NewValue outcomeIndex={outcomeIndex} value={newPrice} />
+              )}
```

Comment:
This is a little ugly, but I want to hide the arrow when the new value is the same.